source/rest/manager/index.php

```diff
@@ -1,118 +1,73 @@
 <?php
 /**
  * CometVisu Manager backend
+ * PHP version 7.4
```

Comment:
bump to PHP 8.0?
Is it relevant here or is the central definition of PHP >= 8.0 sufficient?

Reply:
This is a comment in generated code, changing it manually would get lost on every new generator execution.